.github/scripts/test-nb.sh

```diff
@@ -0,0 +1,10 @@
+set -ae
+
+LATEST_UWTOOLS_VERSION= conda search -c ufs-community --override-channels uwtools | tail -n 1 | grep -Po '[0-9]+\.[0-9]+\.[0-9]'
```

Comment:
Is there a better or easier way to get the latest uwtools version number?

Reply:
Maybe a `conda install uwtools @ git+https://github.com/ufs-community/uwtools@v2.3.2#subdirectory=src` ? That's assuming we already have the conda package available to us, which might be needed `conda install -y -c maddenp -c conda-forge --override-channels condev`. I feel like the `install-conda.sh` script in this directory might be useful since it sets that conda env up. To get the miniforge install there they simply call the url `url=https://github.com/conda-forge/miniforge/releases/download/23.1.0-4/Miniforge3-23.1.0-4-Linux-x86_64.sh`. Could we call to the uwtools repo and download the latest version?

That being said, I'm not entirely sure. @maddenp-noaa might know better what already exists and how we can reuse this to get the latest version of uwtools.

Reply:
I know I answered your question in Slack with "the latest released version", so it makes sense that you'd try to be explicit about it. I think we just get lucky with correct implicit behavior by conda, but that's not necessarily common knowledge.